doc/md/fundamentals/2-types/14-subtyping.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 13
+sidebar_position: 14
 ---
 
 # Subtyping
```

Comment:
The description of invariance is incorrect and should be:
```
Invariant | No subtyping relationship is preserved. | `F[T1]` <: `F[T2]` if, and only if, `T1 <: T2` and `T2 <: T1`  | Since `Nat  <: Int` but not `Int <: Nat`, `[var Nat]` and `[var Int]` are not related by subtyping. You cannot use a mutable array of natural numbers where a mutable array of integers is expected |
```